samples/grids/grid/editing-excel-style-custom/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import 'igniteui-webcomponents-grids/grids/combined';
+import { ComponentRenderer, WebGridDescriptionModule } from 'igniteui-webcomponents-core';
+import { IgcGridComponent } from 'igniteui-webcomponents-grids/grids';
+import { NwindData } from './NwindData';
+
+import "igniteui-webcomponents-grids/grids/themes/light/bootstrap.css";
+
+import "./index.css";
+
+export class Sample {
+
+    private grid1: IgcGridComponent
+    private _bind: () => void;
+
+    constructor() {
+        var grid1 = this.grid1 = document.getElementById('grid1') as IgcGridComponent;
+    
+        this._bind = () => {
+            grid1.data = this.nwindData;
+            grid1.addEventListener('keydown', (event: KeyboardEvent) => {
+                var key = event.key; // Correct variable name used
+                const activeElem = grid1.selectedCells[0];
+
+                if ((key >= '0' && key <= '9') || (key.toLowerCase() >= 'a' && key.toLowerCase() <= 'z')) {
```

Comment:
Comparing strings is unreliable. This would also return true for tab, enter, escape, backspace and so on. You can use keycodes instead like in the angular sample.

Reply:
@tishko0 This is not resolved. As mentioned above special keys will also match the string compare. For example, if you hit Backspace, you will get "Backspace" as text in the cell:
![image](https://github.com/IgniteUI/igniteui-wc-examples/assets/10397980/63481ea8-deeb-47ba-b17a-503856939936)

You need a check that matches only the number and alphabet keys. You can refer to angular's sample for a valid check: 
https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/cell-editing#angular-grid-excel-style-editing-sample

Reply:
Backspace case has been added